migrations/00001_initial/index.sql

```diff
@@ -0,0 +1,22 @@
+DROP SCHEMA IF EXISTS emails CASCADE;
+CREATE SCHEMA emails;
+
+DROP TABLE IF EXISTS emails.signer_emails CASCADE;
+CREATE table emails.signer_emails
+(
+    id            SERIAL PRIMARY KEY,
+    chain_id      int                   NOT NULL,
+    email_address text                  NOT NULL,
+    safe_address  character varying(42) NOT NULL,
+    signer        character varying(42) NOT NULL,
+    verified      boolean               NOT NULL DEFAULT false,
+    UNIQUE (chain_id, email_address, safe_address, signer)
+);
+
+DROP TABLE IF EXISTS emails.verification CASCADE;
+CREATE table emails.verification
+(
+    id                SERIAL PRIMARY KEY REFERENCES emails.signer_emails ON DELETE CASCADE,
```

Comment:
> Is it intended to segment the data vertically?

Can you explain what you mean by this?

Reply:
I mean, the only reason I can imagine for having this kind of 1:1 relationship between these two tables, having the primary keys linked, is to implement some kind of vertical partitioning (aka _row splitting_). Not sure this was the reason, just asking.

If it is not, what's the reasoning for having two separate tables? If `verification_code` and `sent_on` are associated with exactly 1 element of `emails.signer_emails`, why are they separated on a database level?

I think at the _domain level_ these could represent different concepts, but in the _datasouces layer_ they seem to be linked, and perhaps having them linked could simplify `EmailDataSource` because it will operate just one table. 

I don't have an strong opinion on this, but just commenting it out for your consideration 🙂

Reply:
From my perspective, I think the current schema is valid and serves its purpose. But I also think having all this data in the same table would simplify things in the datasources layer, even though additional work might be necessary in the domain layer to split the data into two entities. Merging the columns in a single table will also benefit performance, as fewer queries would be necessary, but I think performance is not a concern for this use case.  So I left this up to you, I'm good with the current design, and even if we think we can improve it, it could be done later in another PR.

Reply:
Overall I agree. I was in between both solutions and decided to split into two table. I do think that we should leverage more the domain layer so if this simplifies our schemas I think it's a good change 👍 . I've merged both tables here: [e6503fb](https://github.com/safe-global/safe-client-gateway/pull/829/commits/e6503fbb11790b97147de898e6e5260875853376)